db/migrate/20150814082503_remove_column_data_from_projects_users_table.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveColumnDataFromProjectsUsersTable < ActiveRecord::Migration
+  def change
+    remove_column :projects_users, :data, :jsonb
```

Comment:
http://guides.rubyonrails.org/association_basics.html#the-has-and-belongs-to-many-association